File: src/main/java/komu/blunt/types/ConstructorNames.java
package komu.blunt.types;

import static com.google.common.base.Strings.repeat;

public class ConstructorNames {

    public static final String CONS = ":";
    public static final String NIL = "[]";
    public static final String UNIT = "()";
    public static final String TRUE = "True";
    public static final String FALSE = "False";

    public static String tupleName(int arity) {
        return "(" + repeat(",", arity-1) + ")";
    }
}

File: src/main/kotlin/komu/blunt/types/ConstructorNames.kt
package komu.blunt.types

import komu.blunt.utils.appendTimes

object ConstructorNames {

    public val CONS: String  = ":"
    public val NIL: String   = "[]"
    public val UNIT: String  = "()"
    public val TRUE: String  = "True"
    public val FALSE: String = "False"

    fun tupleName(arity: Int) = StringBuilder("(").appendTimes(",", arity-1).append(")").toString()
}

